documents/interaction-states.md

```diff
@@ -0,0 +1,51 @@
+# Interaction state machine
+
+During his work, a user can perform different actions, such as moving the mouse, leaving the seat, or typing on the keyboard. Due to the fact that those actions are immediate, it is possible to instantly transition from any state to any other.
+
+Each of the following states is defined in terms of those 3 parameters:
+
+- **Gaze**: workspace(💻), elsewhere (🌲), none (❌);
+- **Voice**: any (❔), present (✔), none (❌);
+- **Input**: any(❔), mouse & keyboard (✔), none (❌).
```

Comment:
I don't think we should be taking into consideration the input from mouse/keyboard, since we're trying to predict engagement from audio/video and **then** correlate that engagement to the keyboard/mouse activity.

Reply:
That was a major source of confusion for me, thanks.